index.html

```diff
@@ -241,5 +256,50 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <section class="form">
+      <p class="form-text">
+        <!-- I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. -->
+      </p>
+      <!-- This is the Form section of the Page with validations -->
+      <form class="form-inputs" action="https://formspree.io/f/xyyoqbgr" method="post">
+        <input type="text" class="form-name" name="name" placeholder="Full name" maxlength="30" required>
+        <input type="email" class="form-email" name="email" placeholder="Email address" required>
+        <textarea name="Message" class="form-comments" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
+        <button type="submit" class="form-button">Get in touch</button>
+      </form>
+      <ul class="list2">
+        <li>
+          <a 
+            href="https://github.com/MarcoAurelioAntonio" 
+            class="github1"></a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/gustavo-porcilio-4496a223a/"
+            class="linkedin1"
+          ></a>
+        </li>
+        <li>
+          <a 
+            href="#" 
+            class="angelist1">
+          </a>
+        </li>
+        <li>
+          <a 
+            href="https://twitter.com/gustavoporcilio" 
+            class="twitter1"></a>
+        </li>
+        <li>
+          <a 
+            href="#" 
+            class="medium1">
+          </a>
+        </li>
+      </ul>
+    </section>
+    <div class="final">
+      <div class="final-line"></div>
+    </div>
```

Comment:
- [ ] Kindly make sure you use the same `size` on all social media icons(images) like on Figma design

- [ ] Kindly make sure your footer styling design sticks to the one from Figma as required for the project e.g.: spacing at the top, missing horizontal line,... Check everything and be sure everything is matching.

| Figma design | Your Design |
| ------------------- | ------------------ |
| ![Screenshot from 2022-07-13 00-12-09](https://user-images.githubusercontent.com/83042742/178605271-5cb47df4-3963-4c5d-8804-94bd98005886.png) | ![Screenshot from 2022-07-13 00-13-09](https://user-images.githubusercontent.com/83042742/178605314-af111fc5-8e6b-4397-8cfd-e5fa50afd83d.png) |